File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/UnitOfMeasure.java
package com.philips.cdp.prxclient.datamodels.specification;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class UnitOfMeasure implements Serializable {

	private static final long serialVersionUID = 8843113320809077214L;
	@SerializedName("unitOfMeasureSymbol")
	@Expose
	public String unitOfMeasureSymbol;

	@SerializedName("unitOfMeasureName")
	@Expose
	public String unitOfMeasureName;

	@SerializedName("unitOfMeasureCode")
	@Expose
	public String unitOfMeasureCode;
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/UnitOfMeasure.kt
package com.philips.cdp.prxclient.datamodels.specification

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class UnitOfMeasure(var unitOfMeasureSymbol: String? = null, var unitOfMeasureName: String? = null, var unitOfMeasureCode: String? = null) : Parcelable